### DISCUSSION TITLE
where do you  find this IP    - http://<YOUR-IP>/webhook

### DISCUSSION THREAD

**Discussion by largalaga** [FabCx/TradingView-Webhook-Bot#72] [General]
hello, thanks for this tool. Question:  how can one discover what IP address to use?

is it the IP address that appears on google when you ask, what is my IP, or is there another IP address that i have to use ? it is not responding or triggering alarts for me. thanks

**Comment by fabston** [comment]
Hi there,

You would have to search for your WAN address however I’d **not** recommend to run it on your local machine. A) Your IP can change B) Its usually not really reliable and C) You’d probably have to open ports. Rather use a commercial VPN. I am using [Hetzner](https://fabston.dev/hetzner).

**Comment by largalaga** [comment]
thanks for the quick reply. Awesome. I was going to setup and test on my local computer before uploading it on my cloud. 

I am using google cloud / AWS  VPS to run of cloud computer. so the IP address i should use is the "External IP". And not the Internal IP?  

I will try that now. thanks.

**Reply by fabston** [reply]
Correct, you would have to use your external IP and be sure to open your ports accordingly.
